src/editor/gridstack/gridstackModel.ts

```diff
@@ -268,26 +272,46 @@ export class GridStackModel {
         cell.appendChild(item.node);
         break;
       }
-      case 'markdown':
-        renderMarkdown({
-          host: cell,
-          source: cellModel.value.text,
-          sanitizer: this.rendermime.sanitizer,
-          latexTypesetter: this.rendermime.latexTypesetter,
-          linkHandler: this.rendermime.linkHandler,
-          resolver: this.rendermime.resolver,
-          shouldTypeset: false,
-          trusted: true
+      case 'markdown': {
+        const markdownCell = new MarkdownCell({
+          model: cellModel as MarkdownCellModel,
+          rendermime: this.rendermime,
+          contentFactory: this.contentFactory,
+          editorConfig: this._editorConfig.markdown,
+          updateEditorOnShow: false
         });
+        markdownCell.inputHidden = false;
+        markdownCell.rendered = true;
+        const coll = markdownCell.node.getElementsByClassName('jp-Collapser');
+        for (let i = 0; i < coll.length; i++) {
+          coll[i].remove();
+        }
+        const pro = markdownCell.node.getElementsByClassName('jp-InputPrompt');
```

Comment:
Small styling thing, maybe this variable could be named `prompt` (so it's explicit):

Reply:
Sure, I'll change it.